source/docs/naming_javascript.md

```diff
@@ -7,7 +7,7 @@ Let's take a look at how this is done.
 
 ### Resolving a Domain to an address
 
-Resolving a domain to its corresponding address is done by calling `EmbarkJS.Names.resolve`, which takes the domain in question as first argument and a callback function that has access to the resolved address as a second argument, and returns a promise that resolves with an address or an error message.
```

Comment:
This last part we're removing wasn't supposed to be there in the first place, as mentioned in: https://github.com/embark-framework/embark-site/pull/76#discussion_r220661730